addons/gunbag/functions/fnc_BIArsenalOpen.sqf

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+/*
+ * Author: mjc4wilton
+ * Handle BI arsenal opening
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_gunbag_fnc_BIArsenalOpen
+ *
+ * Public: No
+ */
+
+if ([player] call FUNC(hasGunBag)) then {
+    GVAR(arsenalCache) = (backpackContainer player) getVariable [QGVAR(gunbagWeapon), []];
```

Comment:
Same as above. Probably does not handle the remote controlled units.

In any case, I would pass around the actual unit instead of relying on global state.